### ISSUE TITLE
[idea] include thresholds when explaining predictions for tree classifiers

### ISSUE THREAD

**Issue by arc12** [TeamHG-Memex/eli5#307]
One thing which would be useful from a "white-boxing" point of view, for tree classifiers, would be an indication of where the decision thresholds* are for a feature. Is the instance value for the prediction data close to the threshold or far from it? This seems quite important for deciding whether or not the feature really is important, or not.

That would be a nice addition.

*-  (as from sklearn Random Forest exposing tree_.threshold)

**Comment by lopuhin** [comment]
I think that's a great idea 👍
